spdlog/src/sink/rotating_file_sink.rs

```diff
@@ -88,7 +91,53 @@ struct RotatorTimePointInner {
     file_paths: Option<LinkedList<PathBuf>>,
 }
 
-/// A sink with a file as the target, rotating according to the rotation policy.
+/// A sink with a collection of files as the target, rotating according to the
+/// rotation policy.
+///
+/// # Overview
+///
+/// A service program that runs for a long time in an environment with limited
+/// hard disk space may continue to write messages to the log file and
+/// eventually run out of hard disk space. `RotatingFileSink` is designed for
+/// such a usage scenario. It periodically removes old log messages and files
+/// from disk to make sure the hard disk won't be filled up by log files.
```

Comment:
Another issue that may not matter is that `RotatingFileSink` should emphasize rotation rather than deletion, so IMO here should mainly describe that the log content will be rotated and split by policy possibly to multiple files, and then just a saying that it is able to delete old files.